app/lib/theme/widgets/RankWidget.tsx

```diff
@@ -0,0 +1,25 @@
+import { WidgetProps } from "@rjsf/core";
+import ReadOnlyCalculatedValueWidget from "./ReadOnlyCalculatedValueWidget";
+
+const RankWidget: React.FC<WidgetProps> = (props) => {
+  return (
+    <>
+      <ReadOnlyCalculatedValueWidget
+        {...props}
+        message={
+          "Enter a project score to see the ranking compared to other scored projects."
+        }
+      />
+
+      <style jsx>{`
+         {
+          dd {
+            margin-bottom: 0;
+          }
+        }
+      `}</style>
```

Comment:
> I don't think this is needed anymore

I'll make a tech debt card for it (I can do it right away; I just want to avoid getting caught up in the sqitch rebasing a few people are doing today)